apps/docs/content/guides/platform/migrating-and-upgrading-projects.mdx

```diff
@@ -93,6 +93,48 @@ If you upgrade to a paid plan while your project is paused, any expired one-clic
   src="/docs/img/guides/platform/paused-paid-tier.png"
 />
 
+#### Restoring a Downloaded Backup Locally
+
+Downloaded backups are plaintext SQL files that can be loaded into a Postgres instance using [psql](https://www.postgresql.org/docs/current/app-psql.html). You can use our open source build tooling if you would like to restore a backup into a local environment so you can interact with it. Here are the steps:
+
+Install the [nix](https://nixos.org/):
```

Comment:
Is "the nix" the right phrasing? Or should it just be "install nix" or perhaps "the nix runtime" (or whatever word other than runtime might be appropriate?)